P/PGPLOT/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PGPLOT"
+version = v"5.2"
+
+sources = [ArchiveSource("ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz", "a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4")]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir pgplot_build && cd pgplot_build/
+cat ../pgplot/drivers.list | sed 's|! PSDRIV|  PSDRIV|g' | sed 's|! GIDRIV|  GIDRIV|g' > drivers.list
+../pgplot/makemake ../pgplot/ linux g77_gcc
+sed -i 's|FCOMPL=g77|FCOMPL=gfortran|' makefile
+make
+cp libpgplot.so $libdir
+install_license ../pgplot/copyright.notice
+"""
+
+platforms = [
+    Linux(:x86_64, libc=:musl),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:i686, libc=:musl),
+    Linux(:i686, libc=:glibc),
+]
+platforms = expand_gfortran_versions(platforms)
```

Comment:
This library doesn't depend on libgfortran as far as I can see
```
% ldd libpgplot.so 
        linux-vdso.so.1 (0x00007ffd0c5d0000)
        libc.so.6 => /usr/lib/libc.so.6 (0x00007f88c6952000)
        /usr/lib64/ld-linux-x86-64.so.2 (0x00007f88c6bcf000)
```
```suggestion
```

Reply:
Ok, but for some reason it requires gfortran in makefile.